src/main/python/protocol/constants.py

```diff
@@ -63,3 +63,5 @@
 # When did we get support for 2-byte macros
 VIAL_PROTOCOL_EXT_MACROS = 5
 VIAL_PROTOCOL_KEY_OVERRIDE = 5
+# When did we indicate optionally-supported features.
+VIAL_PROTOCOL_OPTIONAL_FEATURES = 6
```

Comment:
shouldn't this already be taken care of by the check against VIAL_PROTOCOL_DYNAMIC?

Reply:
I thought you had recently bumped the Vial protocol version from 5 to 6 on the firmware side in the merge-2025-06-24 branch, so I did this to match. But it's not strictly necessary. I revised to just check against VIAL_PROTOCOL_DYNAMIC.